haskell/private/cabal_wrapper.py.tpl

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+
+from glob import glob
+import os
+import os.path
+import re
+import shlex
+import subprocess
+import sys
+import tempfile
+
+def run(cmd, *args, **kwargs):
+    # print("+ " + " ".join([shlex.quote(arg) for arg in cmd]), file=sys.stderr)
+    # sys.stderr.flush()
```

Comment:
I left it intentionally to make future debug easier (since python doesn't have an equivalent to `set -x`). Replaced by a conditional to make it clearer